### ISSUE TITLE
请问能输入一张图片，然后直接输出图片中的内容和相关性吗（也就是不要点击），我想用来给图片生成描述方便搜索

### ISSUE THREAD

**Issue by wacdev** [ttengwang/Caption-Anything#13]
请问能输入一张图片，然后直接输出图片中的内容和相关性吗（也就是不要点击），我想用来给图片生成描述方便搜索

**Comment by ttengwang** [comment]
@wacdev Thanks for the question. We have added the "Caption everything in a paragraph" feature.

**Comment by wanghaisheng** [comment]
@ttengwang  does this "Caption everything in a paragraph" feature rely on openai chatgpt? 
can we use bing gpt instead

**Comment by ttengwang** [comment]
Yes, a chatGPT-like LLM is required for paragraph generation.  It is ok to replace it with another gpt, as long as there is an API available to facilitate the integration.

**Comment by wanghaisheng** [comment]
another question 
without click to drive the prompt, what input gpt consume?
can you add  explanation to existing click driven image like this 
https://github.com/ttengwang/Caption-Anything/blob/main/assets/demo1.png
at last I just want to thank you for your work, this definitely give me confidence and a great start to catch on.
during last 2 years I have dig about Audio description service which I want to integrate a  affordable wearable camera to aid visual impairment people in their daily life.
>Audio description (also referred to as “description” or “video description”) is defined as “the verbal depiction of key visual elements in media and live productions.” AD is meant to provide information on visual content that is considered essential to the comprehension of the program

**Comment by wanghaisheng** [comment]
gotta
@ttengwang

**Comment by shams2023** [comment]
> 请问能输入一张图片，然后直接输出图片中的内容和相关性吗（也就是不要点击），我想用来给图片生成描述方便搜索

请问你成功了吗？我也想直接为图像生成字幕，而不需要什么点击，该如何实现？
